core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -139,6 +154,11 @@ public String getKey() {
         public void setKey(String key) {
             this.key = key;
         }
+        
+        @Override
+        public UserPropertyDescriptor getDescriptor() {
+            return null;
```

Comment:
Not sure how valid this is. Safer to create & register a real descriptor, but return `false` from `isEnabled()`.